src/ui-setup/ui-template/src/utils/index.js

```diff
@@ -1,13 +1,20 @@
 import { render, act } from '@testing-library/react'
 import wait from 'waait'
 
+export const closeTestConductor = (alice, testName) => {
```

Comment:
can we rename `alice` to `agent`?

Reply:
It's been a DNA testing standard to use alice, but I'm fine with making a separate stardard for the UI integration. :+1: